lib/ZuluSCSI_platform_RP2040/audio.h

```diff
@@ -32,6 +32,20 @@ extern "C" {
 // these must be divisible by 1024
 #define AUDIO_BUFFER_SIZE 8192 // ~46.44ms
 
+/*
+ * Status codes for audio playback, matching the SCSI 'audio status codes'.
+ *
+ * The first two are for a live condition and will be returned repeatedly. The
+ * following two reflect a historical condition and are only returned once.
+ */
+enum audio_status_code {
```

Comment:
Would these parts be common to all platforms if another platform gets audio support in future?
If yes, it could be better to make `src/ZuluSCSI_audio.h` for the common parts and interface.

Reply:
Good idea, I'll resolve that.